Node.gitignore

```diff
@@ -24,7 +28,7 @@ coverage
 build/Release
 
 # Dependency directories
-node_modules
+*/node_modules
```

Comment:
I'm not sure this rule is right, as it now shows the root `node_modules` as untracked (rather than ignored):

```
$ npm init
...
$ npm install express
...
$ echo "*/node_modules" > .gitignore
$ less .gitignore
*/node_modules
$ git status
On branch master

Initial commit

Untracked files:
  (use "git add <file>..." to include in what will be committed)

	.gitignore
	Website/
	community-and-safety/
	node_modules/
	package.json
	ruby-hacking/

nothing added to commit but untracked files present (use "git add" to track)
```

With the previous rule, any folder named `node_modules` at any level in the repository will be ignored - I think that's what we're going for here, right?